packages/core/src/Stores/client-store.js

```diff
@@ -94,6 +94,8 @@ export default class ClientStore extends BaseStore {
 
     @observable trading_servers = [];
 
+    @observable last_transaction;
```

Comment:
IMO, we should have this observable in the `portfolio` store since `last_transaction` is related to `portfolio`.